app/models/external_post.rb

```diff
@@ -1,11 +1,8 @@
 class ExternalPost < Post
-	self.table_name = 'external_posts'
+  validates_uniqueness_of :external_id
```

Comment:
Prefer the new style validations `validates :column, uniqueness: value` over `validates_uniqueness_of`.